eng/native/tryrun.cmake

```diff
@@ -144,6 +145,9 @@ elseif(TARGET_ARCH_NAME MATCHES "^(armel|arm|armv6|arm64|loongarch64|riscv64|s39
     set_cache_value(HAVE_SET_MAX_VARIABLE 1)
     set_cache_value(HAVE_FULLY_FEATURED_PTHREAD_MUTEXES 1)
     set_cache_value(HAVE_FUNCTIONAL_PTHREAD_ROBUST_MUTEXES_EXITCODE 0)
+    set_cache_value(HAVE_TERMIOS2_EXITCODE 1)
+  else()
+    set_cache_value(HAVE_TERMIOS2_EXITCODE 0)
```

Comment:
nit: the current style is to add the "default"  before the conditional block (line 103) and override the platform specific values where needed. This way we won't be needing this `else()` block.